modules/articles/client/controllers/articles.client.controller.js

```diff
@@ -1,84 +1,54 @@
 'use strict';
 
-// Articles controller
-angular.module('articles').controller('ArticlesController', ['$scope', '$stateParams', '$location', 'Authentication', 'Articles',
-  function ($scope, $stateParams, $location, Authentication, Articles) {
-    $scope.authentication = Authentication;
+angular
+  .module('articles')
+  .controller('ArticlesController', ArticlesController);
 
-    // Create new Article
-    $scope.create = function (isValid) {
-      $scope.error = null;
+ArticlesController.$inject = ['$scope', '$state', 'Authentication', 'articleResolve'];
 
-      if (!isValid) {
-        $scope.$broadcast('show-errors-check-validity', 'articleForm');
+function ArticlesController($scope, $state, Authentication, articleResolve) {
+  var vm = this;
```

Comment:
The only issue is we can't just change it in one place. Maybe we do that as migration to 1.5?